packer/core_test.go

```diff
@@ -700,6 +678,107 @@ func TestSensitiveVars(t *testing.T) {
 	}
 }
 
+// Normally I wouldn't test a little helper function, but it's regex.
+func testisDoneInterpolating(t *testing.T) {
```

Comment:
Was this supposed to be named as `TestIsDoneInterpolating` to be considered a test? I test it locally and it fails

Reply:
I fixed it using this regex `{{(\\s*|)user(\\s*|)\\x60(.*)\\x60(\\s*|)}}`

Reply:
oh yeah running the test is actually a good idea 🤦‍♀